app/styles/app/modules/navigation.sass

```diff
@@ -69,7 +69,7 @@ $nav-line-height: 35px
     position: absolute
     padding: 0 0 0 0
     margin: -1px 0 0 0
-    z-index: 88
+    @include z-index(above-above)
```

Comment:
Maybe this is wrong?

![image](https://user-images.githubusercontent.com/43280/30702517-7c62146a-9eed-11e7-9839-40caf8f6faab.png)